vpc/subnets/variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "environment_name" { }
+variable "vpc_cidr_block" {
+  default = "10.0.0.0/16"
+}
+variable "vpc_id" { }
+variable "internet_gateway_id" { }
+variable "private_newbits" {
+  default = 8
+}
+variable "private_netnum_offset" {
+  default = 10
+}
+variable "public_newbits" {
+  default = 8
+}
+variable "public_netnum_offset" {
+  default = 0
+}
+variable "az_to_netnum" {
```

Comment:
I dont see any reason to allow someone to overwrite this

Reply:
Is there .a way to lock variables?

Reply:
wooo https://www.terraform.io/docs/configuration/locals.html

Reply:
Does this help? https://www.terraform.io/docs/configuration/locals.html